main.py

```diff
@@ -204,34 +181,161 @@ def create_circuit(self):
 
         return jsonify({"circuit_id": circuit.id}), 201
 
-    def _get_time_from_data(self, data=None):
-        """Receive a dictionary or a string and return a datatime instance.
 
-           data = {
-                 "year": 2006,
-                 "month": 11,
-                 "day": 21,
-                 "hour": 16,
-                 "minute": 30 ,
-                 "second": 00
-           }
+    # METHODS TO HANDLE STOREHOUSE
+    def create_box(self):
+        """Create a new box."""
+        content = {'namespace': self.namespace,
+                   'callback': self._create_box_callback,
+                   'data': {}}
+        event = KytosEvent(name='kytos.storehouse.create', content=content)
+        self.controller.buffers.app.put(event)
+        log.info('Create box from storehouse.')
+
+    def _create_box_callback(self, event, data, error):
+        """Callback to handle create_box."""
+        if error:
+            log.error(f'Can\'t create box with namespace {self.namespace}')
+
+        self.box = data
+        log.info(f'Box {self.box.box_id} was created in {self.namespace}.')
+
+    def list_stored_boxes(self):
+        """List all boxes using the current namespace."""
+        name = 'kytos.storehouse.list'
+        content = {'namespace': self.namespace,
+                   'callback': self._handle_boxes}
+
+        event = KytosEvent(name=name, content=content)
+        self.controller.buffers.app.put(event)
+        log.info(f'Bootstraping storehouse box for {self.namespace}.')
+
+    def _handle_boxes(self, event, data, error):
```

Comment:
What do you say we always use either _callback at the end of the name of each callback names or _handle? Looks like we have a mix of _handle and _callback.